src/Controllers/ShippingController.php

```diff
@@ -55,6 +55,12 @@ public function registerShipments(Request $request, array $orderIds): array {
             'method' => __METHOD__
             ]
         );
+    $this->loggerContract->debug(
+            TranslationHelper::getLoggerKey('debugRegisterShipmentForOrders'), [
+            'additionalInfo' => ['orderIds' => $orderIds, 'request' => $request],
```

Comment:
Have you verified that the `request` object is visible in the logger UI?

Reply:
This was to be removed, I am unable to see the `request object` in the logger UI